config/locales/ja.yml

```
@@ -0,0 +1,28 @@
+ja:
+  activerecord:
+    attributes:
+      book:
+        title: "タイトル"
+        memo: "メモ"
+        author: "著者"
+        picture: "画像"
+  book:
+    heading:
+      index: "書籍一覧"
+      new: "新刊"
+      edit: "編集書籍"
+    action:
+      new: "新刊"
+      show: "詳細"
+      edit: "編集"
+      destroy: "削除"
+      back: "戻る"
+    submit:
+      new: "書籍を新規作成する"
+      edit: "書籍を更新する"
+    message:
+      confirm: "本当によろしいですか？"
+      success:
+        create: "書籍の作成に成功しました。"
```

```suggestion
        create: "%{model}の作成に成功しました。"
```

のようにパラメータを使うことで、再利用可能な翻訳に昇格させることができます。

---

ご指摘があったように、パラメータ"%{model}"を使って再利用できる形に修正しました。